java-compat/README.md

```diff
@@ -199,6 +193,9 @@ for instructions, then follow these steps:
 3. Navigate to and select the "android" subdirectory
 4. Click OK and wait for the project to open and build
 
+To build the tests against the Ice binary distribution you must set `ICE_BIN_DIST` environment
+variable to `all` before starting Android Studio.
```

Comment:
On non-windows os this implies that you have to start Android Studio from the command line right?

Reply:
Not necessarily, you set this in the user environment variables and then start Android Studio

Reply:
I don't think this is really so easy on on macOS or Linux. Probably fine for now though.

Reply:
maybe we should add a Gradle property